cypress.config.js

```diff
@@ -0,0 +1,9 @@
+const { defineConfig } = require("cypress");
+
+module.exports = defineConfig({
+  e2e: {
+    setupNodeEvents(on, config) {
+      // implement node event listeners here
```

Comment:
This content of `setupNodeEvents()` is not being used. Consider removing it. An alternative could be `baseUrl: "http://localhost:3000"` within `e2e` which would allow you to use relative urls for al `cy.visit()` calls.